dali/operators/generic/cast.cc

```diff
@@ -13,38 +13,56 @@
 // limitations under the License.
 
 
-#include "dali/core/static_switch.h"
 #include "dali/operators/generic/cast.h"
+#include "dali/core/static_switch.h"
 
 namespace dali {
 
-template<>
-void Cast<CPUBackend>::RunImpl(SampleWorkspace &ws) {
-  auto &input = ws.Input<CPUBackend>(0);
-  auto &output = ws.Output<CPUBackend>(0);
+template <typename OType, typename IType>
+inline void CPUHelper(OType *out, const IType *in, size_t N) {
```

Comment:
How about killing two birds with one stone and making this name consistent?
```suggestion
inline void CpuHelper(OType *out, const IType *in, size_t N) {
```

Reply:
done